SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialRPCService.h

```diff
@@ -41,7 +41,7 @@ struct EntityRPCType
 	}
 };
 
-struct EntityComponentId
+struct SPATIALGDK_API EntityComponentId
```

Comment:
Is this so that you can put the unit tests in the other module? As this is all stuff that is purely internal I think these should all go in the main gdk module.

Reply:
Yeah sure. We just need to work out where we want thing.

Reply:
@samiwh is right, from the guidelines we've agreed on, if testing code that is not a part of exposed API:
`These tests should go into the Private\Tests directory within the relevant module.
		When an Automation Test matches one-to-one with a particular class, the test file should be named [ClassFilename]Test.cpp,
		e.g. a test that applies only to FText would be written in TextTest.cpp.`

Reply:
So in a new folder here: SpatialGDK/Source/SpatialGDK/Private/Tests ?

Reply:
Also might need test definitions to be duplicated or moved to the core module.

Reply:
Oh, that's a fair point about test definitions. Are we comfortable with moving them to SpatialGDK module (should be sufficient, since I'd expect every other module to include it)?
The folder could be `SpatialGDK/Source/SpatialGDK/Private/Tests/Interop`.